fabric-structure-api-v1/src/main/java/net/fabricmc/fabric/api/structure/v1/FabricStructureBuilder.java

```diff
@@ -113,39 +111,25 @@ private FabricStructureBuilder(Identifier id, S structure) {
 	 *
 	 * <p>This is a required option. Vanilla needs it to function.</p>
 	 *
-	 * @param spacing The average distance between 2 structures of this type along the X and Z axes.
+	 * @param spacing    The average distance between 2 structures of this type along the X and Z axes.
 	 * @param separation The minimum distance between 2 structures of this type.
-	 * @param salt The random salt of the structure. This does not affect how common the structure is, but every
-	 *                structure must have an unique {@code salt} in order to spawn in different places.
-	 *
+	 * @param salt       The random salt of the structure. This does not affect how common the structure is, but every
+	 *                   structure must have an unique {@code salt} in order to spawn in different places.
 	 * @see #defaultConfig(StructureConfig)
 	 */
 	public FabricStructureBuilder<FC, S> defaultConfig(int spacing, int separation, int salt) {
 		return defaultConfig(new StructureConfig(spacing, separation, salt));
 	}
 
 	/**
-	 * Sets the structure configuration which spawns in superflat worlds. If unset, this structure will not spawn in
-	 * superflat worlds.
-	 *
-	 * @see #superflatFeature(FeatureConfig)
+	 * Enables generation of this feature in superflat worlds. Structures will still only generate in biomes
+	 * for which they have been enabled. Superflat worlds will use the plains biome by default.
 	 */
-	public FabricStructureBuilder<FC, S> superflatFeature(ConfiguredStructureFeature<FC, ? extends StructureFeature<FC>> superflatFeature) {
-		Objects.requireNonNull(superflatFeature, "superflatFeature must not be null");
-		this.superflatFeature = superflatFeature;
+	public FabricStructureBuilder<FC, S> enableSuperflat() {
```

Comment:
Imo I would just name this the same as the field name `generateInSuperflat`

Reply:
I want to get this in ASAP so I think im just going to put experimental on this method for now so we can change it later if needed 👍